dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install python3-pip python3 python-is-python3 -y
+RUN pip install --upgrade eth-ape
+RUN ape plugins add ens -y
```

Comment:
It might be nice to actually make this a docker compose project...

We could have all the base plugins automatically added, but then we could have a bunch more commented out the docker compose `.yml` file and then users can plug and play by uncommenting out the plugins they want.

This would also help users make their own images with custom plugins.